### PR TITLE
Fix validate phone number

### PR DESCRIPTION
**Pull request by shaziya5416**

### CODE REVIEW

File: packages/react-web-formo/src/utils/helper.js [hunk 1]
```diff
@@ -242,9 +242,9 @@ export function autoValidate(field, data = {}) {
     if (field.required && isEmpty(field.value)) {
       error = true;
       errorMsg = `${field.label} is required`;
-    } else if (!isEmpty(field.value) && !isValidNumber(field.value)) {
+    } else if (!isEmpty(field.value) && !isValidNumber(field.value) || !validateMobileNumber(field.value)) {
```
Comment (Kkrupanand) [inline packages/react-web-formo/src/utils/helper.js:245]: Why to validate number?

File: packages/react-web-formo/src/utils/helper.js [hunk 2]
```diff
@@ -242,9 +242,9 @@ export function autoValidate(field, data = {}) {
...
       error = true;
-      errorMsg = `${field.label} should be a number`;
+      errorMsg = `Pease enter a valid phone numberS`;
```
Comment (Kkrupanand) [inline packages/react-web-formo/src/utils/helper.js:247]: What is S?

Comment (shaziya5416) [reply packages/react-web-formo/src/utils/helper.js:247]: When I saved using Ctrl + S, the ctrl might not have worked and it printed the S along with it.
I'll change it. Apologies

File: packages/react-web-formo/src/utils/helper.js (hunk 1 above)
Comment (shaziya5416) [reply packages/react-web-formo/src/utils/helper.js:245]: If you want me to remove it just let me know,I'll remove it and push

Comment (Kkrupanand) [reply packages/react-web-formo/src/utils/helper.js:245]: Remove and test

File: packages/react-web-formo/src/utils/helper.js (hunk 2 above)
Comment (Kkrupanand) [reply packages/react-web-formo/src/utils/helper.js:247]: While raising PR you should check the files and changes

**Pull request merged**